### DISCUSSION TITLE
Linux - Packages and Installed folders usage when installing / upgrading

### DISCUSSION THREAD

**Discussion by Thieum** [microsoft/vcpkg#20073] [Q&A] [answered]
In a Linux environment, when I first install a package, the resulting files are located in the `packages ` folder. If I upgrade this package with vcpkg, the files will now be in the `installed ` folder.

Is there any way to control this behavior, to only have one destination, regardless of the operation executed?

Removing / installing a package is a possible workaround at the moment to always have the files in the `packages `folder.

**Comment by autoantwort** [comment, marked as answer]
Normally the packages folder is an implementation detail and should not be used by end users. You should only use the installed folder. If there is some kind of missing copying from the packages folder to the installed folder this is a bug.

**Reply by Osyotr** [reply]
@autoantwort by the way, why is that so? This is basically what we've been doing for 10+ years: every library resides in a separate folder and for each library there is a `.props` file with include directories, lib names, etc. Much like CMake find-scripts.
